hack/test/e2e-aws.sh

```diff
@@ -78,7 +78,7 @@ function setup {
   ## TODO: update to talos-systems once merged
   ${CLUSTERCTL} config cluster ${NAME_PREFIX} \
     --kubeconfig /tmp/e2e/docker/kubeconfig \
-    --from https://github.com/talos-systems/cluster-api-templates/blob/main/aws/standard/standard.yaml > ${TMP}/cluster.yaml
+    --from https://github.com/AlekSi/talos-cluster-api-templates/blob/external-cloud-provider/aws/v0.2/standard/standard.yaml > ${TMP}/cluster.yaml
```

Comment:
Should be changed when https://github.com/talos-systems/cluster-api-templates/pull/5 is merged